src/main/webapp/app/exercises/programming/manage/instructions-editor/analysis/programming-exercise-instruction-analysis.service.ts

```diff
@@ -14,7 +14,7 @@ import {
  */
 @Injectable()
 export class ProgrammingExerciseInstructionAnalysisService {
-    private readonly TEST_CASE_REGEX = new RegExp('.*\\((.*)\\)');
+    private readonly TEST_CASE_REGEX = new RegExp('.*?\\((.*)\\)');
```

Comment:
Why is the `?` needed? Shouldn't `*` match between zero (which covers the `?` case) and infinity times?

Reply:
I guess this would be a reluctant quantifier. But I don't know if this is needed (could be).